extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcProcessor.java

```diff
@@ -260,6 +260,12 @@ public void transform(TransformationContext transformationContext) {
                     // If it declares a scope no action is needed
                     return;
                 }
+                // do not add default scope if some built-in was already added
```

Comment:
Hm, I think that we should just use `CustomScopeAnnotationsBuildItem.isScopeIn(Collection<AnnotationInstance>)` in the code above... but I'm not quite sure if it's a good idea. I mean the javadoc of `AdditionalBeanBuildItem.Builder.setDefaultScope(DotName)` is pretty clear that we add a default scope if no scope declared on the bean class, i.e. we should at least mention annotation transformers as well.

Reply:
Agreed, this is now fixed and pushed